src/psyclone/psyir/nodes/extract_node.py

```diff
@@ -141,6 +151,9 @@ def gen_code(self, parent):
         from psyclone.psyir.tools.dependency_tools import DependencyTools
         dep = DependencyTools()
         self._input_list, self._output_list = dep.get_in_out_parameters(self)
+
+        self.update_vars_and_postname()
```

Comment:
Please add a comment here explaining the purpose of this call.

Reply:
Done.